<?xml version="1.0" ?>
<launch>
	<arg name="resolution" default="0.05" />
	<arg name="depth_levels" default="16" />
	<arg name="num_workers" default="1" />
	<arg name="color" default="true" />
	<arg name="information_distribution" default="false" />
	<arg name="cloud_in" default="/camera/depth/points" />
	<arg name="num_startpoints" default="1000" />
	<arg name="num_scaling_gridpoints" default="10" />

  <node pkg="ufomap_bundled" type="ufomap_server_node" name="ufomap_server_node" output="screen" required="true">
		<remap from="cloud_in" to="$(arg cloud_in)" />
		<param name="num_workers" value="$(arg num_workers)" />
		<param name="resolution" value="$(arg resolution)" />
		<param name="depth_levels" value="$(arg depth_levels)" />
		<param name="color_map" value="$(arg color)"/>
		<param name="information_distribution" value="$(arg information_distribution)" />
		<param name="num_startpoints" value="$(arg num_startpoints)" />
		<param name="num_scaling_gridpoints" value="$(arg num_scaling_gridpoints)" />
  </node>
</launch>
